modules/ztp-ai-install-ocp-clusters-on-bare-metal.adoc

```diff
@@ -0,0 +1,204 @@
+// TELCODOCS-213 4.8 Using Assisted Installer to Install & Manage ocp Clusters on Bare Metal
+//
+//
+//
+
+[id=ztp-ai-install-ocp-clusters-on-bare-metal_{context}"]
+
+.Enabling Assisted Installer Service on Bare Metal
+
+The Assisted Installer Service (AIS) deploys {product-title} clusters. Red Hat Advanced Cluster Management (RHACM) 2.3+ ships with AIS, which is deployed when the MultiClusterHub Operator is enabled on the RHACM hub cluster.
```

Comment:
Don't use a hard coded product version, it will go out of date. Also, FYI, my PR adds two variables for ACM, `{rh-rhacm-first}`, which expands as  "Red Hat Advanced Cluster Management (RHACM)" and `{rh-rhacm}`, which expands as "ACM". I think you should replace and use these instead. My work is not merged yet, is the only problem.